docs/guides/authenticate-cli-to-server.mdx

```diff
@@ -0,0 +1,56 @@
+---
+title: "Connect CLI to Remote Server"
+description: "Connect the Agent Stack CLI to your deployed server"
+---
+
+Use the Agent Stack CLI to configure and manage remote Agent Stack deployments. This guide shows you how to connect your local CLI to a deployed server.
```

Comment:
@jenna-winkler would be nice to explain that agent stack is

client-server architecture, where the CLI is just a client connecting to agentstack server and hence allowing remote access.